integration/tests/runtime-workflow.ts

```diff
@@ -45,28 +50,31 @@ beforeAll(async () => {
     expect(newDB).toMatchSnapshot();
 
     await seedDatabase(knex);
-
-    const pubSub = new PubSub();
-    const runtimeEngine = new GraphbackRuntime(modelText, graphbackConfig);
-    const models = runtimeEngine.getDataSourceModels();
-    const modelSchema = buildSchema(modelText);
-    const services = createKnexPGCRUDRuntimeServices(models, modelSchema, knex, pubSub);
-    const runtime = runtimeEngine.buildRuntime(services);
-
-    const server = new ApolloServer({
-      typeDefs: printSchema(runtime.schema),
-      resolvers: runtime.resolvers,
-      context: services
-    });
-
-    client = createTestClient(server);
+    
+    runtimeEngine = new GraphbackRuntime(modelText, graphbackConfig);
+    modelSchema = buildSchema(modelText);
+   
     db = knex;
   } catch (e) {
     console.log(e);
     throw e;
   }
 })
 
+beforeEach(() => {
+  const services = createKnexPGCRUDRuntimeServices(runtimeEngine.getDataSourceModels(), modelSchema, db, new PubSub());
+  const runtime = runtimeEngine.buildRuntime(services);
+  server = new ApolloServer({
+    typeDefs: printSchema(runtime.schema),
```

Comment:
`printSchema` actually removes directives from the schema, so if our tests have them in future this won't validate them.

You could replace this with

```suggestion
schema: runtime.schema,
```

I haven't tested the implications of this, but it should work. If we still need to use typeDefs you can use this `printSchemaWithDirectives`:
 https://github.com/aerogear/graphback/blob/2c72ddb03058a6b5c478fe28067325aebd172d5e/packages/graphback-core/src/utils/printSchemaWithDirectives.ts#L4

Reply:
I did not change the way schema are passed around, but it is a solid suggestion which I have applied, let's see what CI will say :-)